tests/__tests__/sitka.test.ts

```diff
@@ -1,17 +1,97 @@
+import { Dispatch } from "redux"
+import { createSitkaAndStore } from "../sitka-test"
+import { createAppStore, Sitka } from "../../src/sitka"
+
+export class SitkaMock<T = {}> extends Sitka {
+  public registeredModules: T
+ }
+
 describe("Sitka", () => {
 
-  test(`setDispatch dispatch was called`, () => {
+  describe('setDispatch', () => {
+    // Setup
+    const { sitka, store } = createSitkaAndStore()
+    let wasMockDispatched: boolean
+    beforeEach(() => {
+      wasMockDispatched = false
+    })
+
+    test(`setDispatch dispatch was called`, () => {
+      const mockDispatch: Dispatch = <T>(action: T) => {
+        wasMockDispatched = true
+        store.dispatch(action)
+        return action
+      }
+
+      sitka.setDispatch(mockDispatch)
+      sitka.getModules().text.handleUpdateSize(20)
+
+      const { text: actual } = store.getState()
```

Comment:
this test doesn't need to care about actual versus expected state. it only needs to care about whether or not your match dispatch was called